test/integration.test.ts

```diff
@@ -354,7 +354,9 @@ describe("Integration Test", function() {
         for (const batchID of stakedBatchIDs) {
             const preBalance = await coordinator.getBalance();
             const tx = await rollup.connect(coordinator).withdrawStake(batchID);
-            const txFee = (await tx.wait()).gasUsed.mul(tx.gasPrice);
+            const txReceipt = await tx.wait();
+            assert.isDefined(tx.gasPrice);
+            const txFee = txReceipt.gasUsed.mul(tx.gasPrice ?? -1);
```

Comment:
Is there a better default value we can use asides from `-1`? We should never hit this case with the `assert.isDefined` above but it's needed for TypeScript.

Reply:
I used to write `if (tx.gasPrice === undefined) throw Error()` and then TypeScript doesn't complain the next line.
Writing `-1` sounds fine to me. If we really worried we can comment something like "write -1 here to make ts happy".